src/object-specs.ts

```diff
@@ -154,18 +170,19 @@ function valueObjectConfigPathFuture(requestedPath:File.AbsoluteFilePath, config
   return absoluteValueObjectConfigPath;
 }
 
-export function generate(directoryRunFrom:string, parsedArgs:CommandLine.Arguments):Promise.Future<WriteFileUtils.ConsoleOutputResults> {
+export function generate(directoryRunFrom:string, extension:ObjectSpecExtension, parsedArgs:CommandLine.Arguments):Promise.Future<WriteFileUtils.ConsoleOutputResults> {
```

Comment:
Not in love with the concept that this code has logic based on the extension. I'd rather we pass in the default includes/excludes set to apply into the found types. This makes it more flexible if we want to support different object types in the future. We also probably want to disable the equality, code, etc plugins for the plain object specs